libnavigation-core/src/test/java/com/mapbox/navigation/core/history/MapboxHistoryRecorderTest.kt

```diff
@@ -23,23 +21,9 @@ class MapboxHistoryRecorderTest {
     private val logger: Logger = mockk(relaxUnitFun = true)
 
     @Test
-    fun `historyRecorder fileDirectory is null by default`() {
```

Comment:
We should keep this test. Is the default behavior changing? This test verifies that it does not crash with default options

Reply:
test returned back, but now default path is not null, it is calculated in `HistoryFiles`
(it is always not null, because there is no `enabled` param anymore)

Reply:
👍  yeah that makes sense. also 👍  on the new test, thanks for adding!